cmd/installer/cmd/image.go

```diff
@@ -83,6 +85,16 @@ func runImageCmd() (err error) {
 		},
 	}
 
+	if options.ConfigSource == "" {
+		switch p.Name() {
+		case "aws", "azure", "digital-ocean", "gcp":
+			options.ConfigSource = "none"
```

Comment:
We should really rename this. Maybe `default` instead. Something for the future.